Bedrock/NSDate+Bedrock.h

```diff
@@ -10,10 +10,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_OPTIONS(NSInteger, NSTimeIntervalUnit) {
+	NSTimeIntervalUnitHour = (1 << 1),
+	NSTimeIntervalUnitMinute = (1 << 2),
+	NSTimeIntervalUnitSecond = (1 << 3),
+	NSTimeIntervalUnitMillisecond = (1 << 4),
+};
+
+extern NSTimeInterval NSTimeIntervalWithMilliseconds(double milliseconds);
 extern NSTimeInterval NSTimeIntervalWithMinutes(double minutes);
 extern NSTimeInterval NSTimeIntervalWithHours(double hours);
 extern NSTimeInterval NSTimeIntervalWithDays(double days);
 
+extern NSString * NSStringFromNSTimeInterval(NSTimeInterval timeInterval, NSTimeIntervalUnit units);
```

Comment:
Looking at the implementation of this function, it seems to replicate what `DateComponentsFormatter` does. I'd suggest keeping it (and `NSTimeIntervalUnit` enum) specific to The O – this code should ideally be replaced by that built-in class, so "blessing" it by moving it into Bedrock doesn't seem appropriate.